libs/androidlib/package.mill

```diff
@@ -86,16 +86,16 @@ object `package` extends MillPublishScalaModule with BuildInfo {
       Deps.androidDataBindingCompiler,
       Deps.androidDataBindingCompilerCommon,
       Deps.osLib,
+      mvn"org.jetbrains.kotlin:kotlin-stdlib:1.9.10"
```

Comment:
Should be defined in `Deps`. About the version, is this the lowest version for the API?